.github/workflows/buildmgr.yml

```diff
@@ -14,6 +14,7 @@ on:
       - 'libs/rtemodel/**'
       - 'libs/rtefsutils/**'
       - 'tools/buildmgr/**'
+      - '!**/*.md'
```

Comment:
Though it doesn't make much difference, would it be better to separate the include and exclude paths?
```
on:
  push:
    paths-ignore:
      - **/*.md
```

Reply:
Afaik, `paths` and `paths-ignore` are mutually-exclusive.

Reply:
Ahh I see, 

`You can not use paths and paths-ignore to filter the same event in a single workflow. If you want to both include and exclude path patterns for a single event, use the paths filter along with the ! character to indicate which paths should be excluded.`